src/cow-react/modules/limitOrders/pure/Orders/index.tsx

```diff
@@ -115,6 +116,7 @@ const ExternalArrow = styled.span`
 export interface OrdersProps extends OrdersTabsProps, OrdersTableProps {
   isWalletConnected: boolean
   isOpenOrdersTab: boolean
+  children?: ReactNode
```

Comment:
What about `PropsWithChildren` instead?

Example usage in the project https://github.com/cowprotocol/cowswap/blob/98ce5deee6ab62f1209a7f03a4cc4ac0bf7fdb32/src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/Step.tsx#L38

Reply:
It's not needed now, just deleted it, thanks!